lib/cli/src/generators/WEB-COMPONENTS/index.js

```diff
@@ -15,17 +14,18 @@ export default async (npmOptions, { storyFormat }) => {
     '@storybook/web-components',
     '@storybook/addon-actions',
     '@storybook/addon-links',
+    '@storybook/addons',
```

Comment:
Is `@storybook/addons` really needed directly as it's already a dep of `@storybook/addon-actions` and `@storybook/addon-links`? Maybe @ndelangen have some info about that

Reply:
If it works without, let's remove it!

Reply:
@tooppaaa the other points have been resolved and will be done in a different PR, but can you check this one? then we're good to merge!

Reply:
https://github.com/storybookjs/storybook/pull/10703#issuecomment-628227892